src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -315,6 +315,8 @@ void Engine::GetPerformanceStreams(std::map<std::string, std::string>& config,
     config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = common_hints.first;
     config[ov::num_streams.name()] = common_hints.first;
     config[CONFIG_KEY(CPU_THREADS_NUM)] = std::to_string(common_hints.second.num_threads);
+    config[CONFIG_KEY_INTERNAL(CPU_THREADS_PER_STREAM)] =
+        std::to_string(std::max(1, common_hints.second.num_streams / std::max(1, common_hints.second.num_threads)));
```

Comment:
why is it required for OMP specifically?

Reply:
`CPU_THREADS_PER_STREAM` used by legacy code for OMP. 

We are using `THREADS_PER_STREAM_BIG` and `THREADS_PER_STREAM_SMALL` for TBB now.

Reply:
Should be `std::to_string(std::max(1, common_hints.second.num_threads / std::max(1, common_hints.second.num_streams)));` ?

Reply:
Sorry, the divisor and dividend are reversed.